### DISCUSSION TITLE
Change the cursor color

### DISCUSSION THREAD

**Discussion by NetCzech** [stefankueng/BowPad#246] [Ideas]
Hello,
I would like to ask if it is possible to change the color of the cursor on a dark topic. The cursor is now black and almost invisible.
Thank you

![cursor](https://user-images.githubusercontent.com/76791894/158032460-dccd56f2-435b-445c-b950-52fa94b6e6c7.png)

Thank you very much for your work

**Comment by SbiriJJ** [comment]
Same problem :(

**Comment by stefankueng** [comment]
open the settings file, then find the `[view]` section and add or modify the `d2d `entry:
```
[View]
d2d=0
```
this disables the Direct2D drawing. If your cursor is after this properly visible, then that means you should update your graphic cards driver.

**Comment by ags1234** [comment]
Same question here: how can the cursor color be changed? I want to make it a very distinguishable color.
I also want to thank you very much for your work.
